packages/@dcl/ecs/src/components/definitions/OnPointerResult.proto

```diff
@@ -0,0 +1,17 @@
+﻿syntax = "proto3";
+
+import "common/id.proto";
+option (ecs_component_id) = 1065;
+import "common/Vector3.proto";
+
+message PBOnPointerResult {
+  string identifier = 1;
+  int64 entity_id = 2;
```

Comment:
why entity_id ?
This could be on the wireMessage, like all the other components.
```ts
const result = PointerResult.getFrom(entity)
```

Reply:
Removed!